packages/templates/api/assemblyscript/src/__tests__/e2e/integration.spec.ts

```diff
@@ -0,0 +1,111 @@
+import { Web3ApiClient } from "@web3api/client-js";
+import {
+  buildAndDeployApi,
+  initTestEnvironment,
+  stopTestEnvironment,
+  runCLI,
+} from "@web3api/test-env-js";
+import path from "path";
+
+import { getPlugins } from "../utils";
+
+jest.setTimeout(500000);
+
+describe("SimpleStorage", () => {
+  const CONNECTION = { node: "http://localhost:8545" };
+
+  let client: Web3ApiClient;
+  let ensUri: string;
+  let app: any;
```

Comment:
Let's avoid the use of any types. We can simply import the types at the top of the file like so `import * as App from "../types/w3"` which would retain type information.

Reply:
Ahh, apologies, I now see why you chose to do this. It's because you're generating the types in-line in your beforeAll step.

IMO having the types be cast as `any` defeats the purpose of the types, so we should take this out of the test's run-time and instead do this as a pre-processing step, allowing us to import types normally.